dotnet-datetime/StopWatchCSharp/StopWatchCSharp/Program.cs

```diff
@@ -0,0 +1,11 @@
+﻿using StopWatchCSharp;
+using System.Diagnostics;
+
+var stopWatch = StopWatchMethods.CreateRandomArray(1000);
+
+Console.WriteLine($@"Elapsed Time: {stopWatch.Elapsed}
+Elapsed Milliseconds: {stopWatch.ElapsedMilliseconds}
+Elapsed Ticks: {stopWatch.ElapsedTicks}
+Is Running: {stopWatch.IsRunning}
+Frequency: {Stopwatch.Frequency}
+Using High Resolution: {Stopwatch.IsHighResolution}");
```

Comment:
Why are these two `Stopwatch` instances starting with an uppercase letter why all of the previous instances are spelled like `stopWatch`?

Reply:
@mahrncic Those are static read-only fields that are part of the Stopwatch class: https://learn.microsoft.com/en-us/dotnet/api/system.diagnostics.stopwatch.frequency?view=net-7.0. Therefore, we use Stopwatch.IsHighResolution to fetch its value.